news/2023/2023-02-09-Skin-of-the-Year-2022.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Skin of the Year 2022: Voting Open"
+date: 2023-02-09 09:00:00 +0000
+---
+
+Do you think you got what it takes to select the BEST skins of 2022? Let us know by casting your vote!
+
+![](/wiki/shared/news/2023-02-01-Skin-of-the-Year-2022/SOTY-2022-banner.jpg)
+
+Four score and seven years ago (that being 2007), skinning was first introduced to the then newly released EBA simulator called osu!. Ever since, a sprawling and ever so creative community has been constantly expanding and evolving, many thousands of skins have been uploaded during the past 15 years, many styles and iterations of skins have come and gone, but the community never had a "Skin of the Year" competition. Instead of taking a moment to ponder at why that is, let's instead focus on the actual voting in this first-of-a-kind skin competition!
+
+## Voting details
+
+All skins that have been released during 2022 and added to [the compendium](https://compendium.skinship.xyz/) are eligible to compete, **including contest & podium entries**. The top-voted entry will receive a unique profile badge based on their entry and 3 months of osu!supporter. Entries #2 – #10 will receive 1 month of osu!supporter.
+
+You might think "why would I bother voting when the [YUGEN](https://osu.ppy.sh/community/forums/topics/365036?n=1) is the best?", and while that skin is really cool, we urge you to check out the sheer amount of creativity that various artists have put into their works throughout the year with **290** skins that have been submitted to the [skinning forums](https://osu.ppy.sh/community/forums/15) and **136** skins submitted to the compendium, all of them waiting to grab your attention, and maybe even become your main skin.
```

Comment:
```suggestion
You might think "why would I bother voting when [YUGEN](https://osu.ppy.sh/community/forums/topics/365036?n=1) is the best?", and while that skin is really cool, we urge you to check out the sheer amount of creativity that various artists have put into their works throughout the year with **290** skins that have been submitted to the [skinning forums](https://osu.ppy.sh/community/forums/15) and **136** skins submitted to the compendium, all of them waiting to grab your attention, and maybe even become your main skin.
```

"when the YUGEN"? Is this intentionally referring to a skin as a person or something? If not, remove the `the`.

Also in general the sentence reads bad with the "and" and commas proceeding the quoted portion.

Reply:
Looks like `the` was left on accident, I removed that now. As for the sentence itself I tried changing it via https://github.com/ppy/osu-wiki/pull/8773/commits/2a566039435bc460fb51056c4f7d4e87dbaa43dc, but I'll keep this open for review since I am unsure if this was the best approach